### DISCUSSION TITLE
Custom advanced module (not related with inventory) ?

### DISCUSSION THREAD

**Discussion by apcloic** [YetiForceCompany/YetiForceCRM#14820] [How to] [answered]
Hi,

I need to create a custom advanced module that won't be related to inventory module at all.
So, at first I've created a new custom module and choose advanced module type.
And then I've created 2 fields in basic line block.
When creating a record on my custom module, there is no advanced block. 
Do I need to do something else in order to create an advanced module ?
Advanced module are limited to some kind of "inventory" module type ?

![yeti_adv_module_creation gif](https://user-images.githubusercontent.com/2930560/121538895-72868d80-ca05-11eb-8a80-ecc4394ef625.gif)

Regards

**Comment by vovpff** [comment, marked as answer]
You need to change module type and add some advanced fields

**Reply by apcloic** [reply]
@vovpff thanks but module type is already set as advanced and I've added 2 advanced fields as you can see on the video recorded at gitdeveloper.yetiforce.com.
Is there anything else to do?

**Reply by vovpff** [reply]
You need to add `Item Name` advanced field. This is critical

![image](https://user-images.githubusercontent.com/8545791/121650749-8f24d300-caa2-11eb-83ba-678cc0d539d6.png)

**Reply by apcloic** [reply]
@vovpff That's it, adding Item name field did the trick.
Thanks a lot !
